src/discussions/learners/LearnersContentView.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useState } from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router';
+
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import {
+  Avatar, Button, ButtonGroup, Card, Form, Icon, IconButton, Spinner,
+} from '@edx/paragon';
+import { MoreHoriz, Report } from '@edx/paragon/icons';
+
+import { RequestStatus } from '../../data/constants';
+import Comment from '../comments/comment/Comment';
+import { Post } from '../posts';
+import {
+  learnersLoadingStatus, selectCommentsOfUser, selectLearner,
+  selectLearnerAvatar,
+  selectPostsOfUser, selectResponsesOfUser,
+} from './data/selectors';
+import { fetchUserComments, fetchUserPosts } from './data/thunks';
+import messages from './messages';
+
+export const TABS = {
+  POSTS: 'posts',
+  COMMENTS: 'comments',
+  RESPONSES: 'responses',
+};
+
+function LearnersContentView({ intl }) {
+  const [tab, setTab] = useState(TABS.POSTS);
+  const { courseId, learnerUsername: userId } = useParams();
+  const dispatch = useDispatch();
+  const comments = useSelector(selectCommentsOfUser(userId));
+  const posts = useSelector(selectPostsOfUser(userId));
+  const responses = useSelector(selectResponsesOfUser(userId));
+  const apiStatus = useSelector(learnersLoadingStatus());
+  const learner = useSelector(selectLearner(userId));
+  const avatar = useSelector(selectLearnerAvatar(userId));
+
+  useEffect(() => {
+    dispatch(fetchUserPosts(courseId, userId));
+    dispatch(fetchUserComments(courseId, userId));
+  }, [courseId, userId, tab]);
+
+  const handleTabChange = (val) => setTab(val);
+
+  return (
+    <div className="learner-content d-flex flex-column">
+      <Card>
+        <Card.Body>
+          <div className="d-flex flex-row align-items-center m-3">
+            <Avatar src={avatar} alt={userId} />
+            <span className="font-weight-bold mx-3">
+              {learner.name}
+            </span>
+            <div className="ml-auto">
+              <IconButton iconAs={Icon} src={MoreHoriz} alt="Options" />
+            </div>
+          </div>
+        </Card.Body>
+        <Card.Footer className="pb-0 bg-light-200">
+          <div className="d-flex flex-row w-100 my-2">
+            <div className="spacer w-25" />
+            <div className="w-50 text-center">
+              <ButtonGroup>
+                <Button data-testid={`${TABS.POSTS}-btn`} variant="primary" className={tab !== TABS.POSTS ? 'bg-white border-gray-100 text-primary' : ''} onClick={() => handleTabChange(TABS.POSTS)}>
+                  {intl.formatMessage(messages.postsTab)} <span className="ml-3">{posts.length}</span>
+                  <span className="ml-3">
+                    <Icon src={Report} />
+                  </span>
```

Comment:
This should only show up if there is reported content.

Reply:
The report icon shows up only when there are active flags for a user. Extra tests have been added to verify this.